src/api/common_api.js

```diff
@@ -1342,6 +1342,9 @@ module.exports = {
                 },
                 assume_role_policy: {
                     $ref: '#/definitions/assume_role_policy'
+                },
+                delete: {
```

Comment:
Although it was my suggestion, I think we shuldn't add the `delete` here 🙂  
The `role_config` is also used in the `account_schema` and it might cause a confusion. 
Maybe better to add a `delete_role_config` flag in the `update_account` params as you initially suggested